boards/openlabs-kw41z-mini/doc.txt

```diff
@@ -3,20 +3,23 @@
 @ingroup     boards
 @brief       Support for openlabs-kw41z-mini
 
- ### General information
+### General information
 
 This is an open-source development board shipped by openlabs.co with
 source files located at https://openlabs.co/OSHW/kw41z-mini
 
 A driver for the radio transceiver is available in #12277 (802.15.4 only).
 
+@note A version of this board exists that comes with the `M41W8VT4` (256k ROM)
+instead of the `M41W9VT4` (512k ROM). The boards are otherwise identical.
+See @ref boards_openlabs-kw41z-mini-256kib if you happen to have this version.
 
 ### Programming Pinout
 
 Programming and debugging can be done with a Raspberry Pi (or equivalent)
 running OpenOCD.
 
-
+~~~~~~~~~
     Pi GPIO Pins       kw41z-mini
```

Comment:
Not sure why, but in the rendered version `GPIO` now has a link to [here](https://doc.riot-os.org/cc26xx__cc13xx__gpio_8h.html#a1037b18e2d226fe7d327d4a6f17a21c1) on my machine (doxygen v1.8.13, so same used to render the online doc), maybe use indentation after all? With that I don't have the problem at all.

Reply:
Same problem with ```. Only indentation seems to fix that.

Reply:
Alright, indented it.